common/core/gpudevice.cpp

```diff
@@ -201,6 +201,8 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
           "UpdateDisplayState.");
       UpdateDisplayState();
     }
+    /* Adding sleep to avoid busy looping before a retry */
+    usleep(1000);
```

Comment:
I think lseek() will help when its really big buffer where you are reading in same buffer multiple times or in a file. Here I think its small number of bytes, which we read and parse until we find both strings (drm evnent and hot plug). If you add lseek here I think you are just adding one more system call.